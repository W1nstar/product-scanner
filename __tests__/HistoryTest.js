import React from 'react';
import History from '../Components/History';

import renderer from 'react-test-renderer';

test('History renders correctly', () => {
    const navigation = { addListener: jest.fn() };

    const tree = renderer.create(<History navigation={navigation}  />).toJSON();
    expect(tree).toMatchSnapshot();
});