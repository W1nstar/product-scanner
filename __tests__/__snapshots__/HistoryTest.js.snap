// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Lobster-Regular",
        "fontSize": 50,
      }
    }
  >
    Oups...
  </Text>
  <Text
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Vous n'avez pas encore scanné de produit ! Commencez par scanner un produit depuis l'écran d'accueil. Vous retrouverez tous vos scans ici !
  </Text>
</View>
`;
